### ISSUE TITLE
Half Map not clickable after screen orientation change

### ISSUE THREAD

**Issue by pncosta** [ionic-team/ionic-native-google-maps#89]
**I'm submitting a ...**  (check one with "x")
- [ ] question
- [x] any problem or bug report
- [ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
- [x] **Android**
- [ ] **iOS**
- [ ] **Browser**

**cordova information:**  

```
com.googlemaps.ios 2.7.0 "Google Maps SDK for iOS"
cordova-plugin-advanced-http 1.11.1 "Advanced HTTP plugin"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-globalization 1.11.0 "Globalization"
cordova-plugin-googlemaps 2.3.6 "cordova-plugin-googlemaps"
cordova-plugin-ionic-keyboard 2.0.5 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 1.1.19 "cordova-plugin-ionic-webview"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
```

**If you use `@ionic-native/google-maps`, please tell the package.json (only `@ionic-native/core` and `@ionic-native/google-maps` are fine mostly)**
```
 "@angular/core": "5.2.11",
"@ionic-native/google-maps": "^4.14.0",
 "cordova-plugin-googlemaps": "^2.3.6",
```
**Current beahavior:**

I load the app with a page containing only a map in portrait mode. 
I can navigate on the map (drag, zoom, etc) as expected.

When I rotate the screen to landscape mode only the left part of the map is clickable (for drag zoom, etc).
All clicks on the right side of the map are ignored.

If I open and close the hamburger menu, then the hole map becomes clickable.

Likewise, If I start the app on landscape mode and rotate to portrait, the bottom area is not clickable.

I guess the "clickable" left area corresponds to the initial width on portrait mode and for some reason the map is not being correctly drawn after the screen rotation?

**Expected behavior:**

after rotating the screen, the whole map should be clickable (for drag, zoom, etc)

**Screen capture or video record:**

Example on android  : https://youtu.be/zrVJotPrA9Y 

**Related code, data or error log (please format your code or data):**
 
project: https://github.com/pncosta/Lisboa-Limpa

**Comment by wf9a5m75** [comment]
Umm, :(
I will check it later.

Thank you for sharing the project in advance.

**Comment by wf9a5m75** [comment]
ping

**Comment by wf9a5m75** [comment]
Since there is no response for a while, I close this thread.

**Comment by pncosta** [comment]
first, thanks for the response :)

sorry, im on holidays and without much internet :)

i know it is now closed but well, here goes my feedback:

i didnt understand your example as you only rotate the screen but you never tried to drag/pan/zoom the map?

the problem I reported is that half of the map is not clickable/dragable after the screen rotates.


 for now I was able to get a temporary hack, by listening when the screen rotates and calling `this.map = GoogleMaps.create('map_canvas', {});` (im just afraid if this is leading to mem leaks , i still havent really tested it.
